test/Constraints/tuple_arguments.swift

```diff
@@ -1626,7 +1626,7 @@ func rdar33239714() {
 do {
   func foo(_: (() -> Void)?) {}
   func bar() -> ((()) -> Void)? { return nil }
-  foo(bar()) // expected-error {{cannot convert value of type '((()) -> Void)?' to expected argument type '(() -> Void)?'}}
+  foo(bar()) // Allow ((()) -> Void)? to be passed in place of (() -> Void)? for -swift-version 4 but not later.
```

Comment:
Can you make a Swift 5 test that shows it failing?

Reply:
Yes. I considered that earlier today, wondering what the “right time” would be to fork this test again for Swift 5. Perhaps now is that time.